src/apps/backend/modules/booking/types.ts

```diff
@@ -0,0 +1,35 @@
+export class PhoneNumber {
```

Comment:
Is there a reason why this is a class (and some other typedef such as `Address` and `ScheduleTime`, while 90% of the other types are interface?